client/FxAccountClient.js

```diff
@@ -1452,6 +1452,22 @@ define([
   };
 
   /**
+   * Check to see if the secondary email feature is enabled for this user.
+   *
+   * @method recoveryEmailSecondaryEmailEnabled
+   * @param {String} sessionToken SessionToken obtained from signIn
+   */
+  FxAccountClient.prototype.recoveryEmailSecondaryEmailEnabled = function (sessionToken) {
+    required(sessionToken, 'sessionToken');
```

Comment:
Can you pop this call to `required` inside the promise chain, so that it rejects rather than throws?